src/forceUpdatedComponent.js

```diff
@@ -0,0 +1,41 @@
+import React, { PureComponent } from 'react'
+import { wrapDisplayName } from 'recompose'
+
+import { subscribe, unsubscribe } from './pubsub.js'
+
+/**
+ * Returns a HOC that triggers a re-render of itself whenever
+ * a subscription notification is received.
+ */
+export default function forceUpdatedComponent(InputComponent) {
+  class SubscribeUpdater extends PureComponent {
+    constructor(props) {
+      super(props)
+
+      this.state = {
```

Comment:
I think we have class properties enabled in babel so we can change this to
```
state = {
  lastNotification: null,
}
```
and remove the constructor completely